server/types.ts

```diff
@@ -51,6 +51,15 @@ export type Club = {
   content: clubContent;
 };
 
+export type eventContent = {
+  date: string;
+  period: string;
+  type: string;
+  topic: string;
+  description: string;
+  rsvpLink: string;
+};
+
```

Comment:
I don't think this `eventContent` type is used anywhere in the code, so it can be deleted.